app/views/layouts/application.html.slim

```diff
@@ -61,11 +61,12 @@ html.govuk-template.app-html-class lang="en"
 
       main.govuk-main-wrapper#main-content role="main"
         - unless flash.empty?
```

Comment:
Can we get rid of this line? If `flash` is empty, does `flash.each` error, or do nothing?